docs/index.md

```diff
@@ -110,6 +110,17 @@ $ retis pcap --probe tp:net:net_dev_start_xmit -o retis.pcap
 $ wireshark retis.pcap
 ```
 
+Some post-processing commands (eg. `print`, `sort`) can generate a long outputs.
```

Comment:
I have only this tiny nit for this version: "a long output" or "long outputs".

Other than that, the PR can be merged, AFAIC.